src/julia.expmap

```diff
@@ -32,7 +32,7 @@
     utf8proc_*;
     jlbacktrace;
     _IO_stdin_used;
-    llvm_to_jl_value;
+    __ZN4llvm23createLowerSimdLoopPassEv;
```

Comment:
ok, but that is still only a draft, and it only applies to amd64 (http://mentorembedded.github.io/cxx-abi/). we have many `extern "C" wrappers for this reason.

(although not really relevant now, i also don't immediately see why this has been inserted into the llvm namespace. the llvm documentation examples suggest putting new passes in the anonymous namespace http://llvm.org/docs/WritingAnLLVMPass.html)

also unrelated, but it seems the LowerSIMDLoop pass should implement `getAnalysisUsage` so that LLVM doesn't need to invalidate the CFG afterwards.